controllers/profiles/common/properties/application.go

```diff
@@ -75,36 +75,34 @@ func (a *appPropertyHandler) WithServiceDiscovery(ctx context.Context, catalog d
 }
 
 func (a *appPropertyHandler) Build() string {
-	var props *properties.Properties
+	var userProps *properties.Properties
```

Comment:
Related to the comment I did before, this ApplicationPropertyHandler has changed the original semantic. Maybe we should rename and update a bit the documentation of the function NewAppPropertyHandler.